tests/integration/features/steps/integration_steps.py

```diff
@@ -1482,7 +1487,7 @@ def write_dummy_file(path, mtime_str, contents=None):
 
 def get_mgmt_api_jars(
         version,
-        url="https://api.github.com/repos/datastax/management-api-for-apache-cassandra/releases/latest"):
+        url="https://api.github.com/repos/k8ssandra/management-api-for-apache-cassandra/releases/latest"):
```

Comment:
Thought:
Wow, this shows how long it has been since I visited this. C* 4.1 has come out since and we moved the Management API project to a different GitHub organization since the last time I touched this!